history-network.md

```diff
@@ -130,6 +130,17 @@ _MAX_HEADER_LENGTH = 2**13  # = 8192
 MAX_ENCODED_UNCLES_LENGTH = _MAX_HEADER_LENGTH * 2**4  # = 2**17 ~= 131k
 # Maximum number of uncles is currently 2. Using 16 leaves some room for the
 # protocol to increase the number of uncles.
+
+MAX_WITHDRAWAL_COUNT = 16
+# Number sourced from EIP-4895
```

Comment:
I don't actually noticed that limit in EIP-4895, it is however enforced in consensus mainnet preset: https://github.com/ethereum/consensus-specs/blob/f7352d18cfb91c58b1addb4ea509aedd6e32165c/presets/mainnet/capella.yaml#L12

Perhaps link to that and perhaps set `MAX_WITHDRAWAL_COUNT = MAX_WITHDRAWALS_PER_PAYLOAD`